.github/workflows/python.yaml

```diff
@@ -73,5 +81,9 @@ jobs:
         run: poetry run pylint $(git ls-files '*.py')
 
       - name: Run tests
-        if: ${{ inputs.testsMustPass }}
+        if: ${{ inputs.testsMustPass && inputs.codeCoverageFailUnder == 0 }}
         run: poetry run pytest
+
+      - name: Run tests and compare code coverage
+        if: ${{ inputs.testsMustPass && inputs.codeCoverageFailUnder != 0 }}
+        run: poetry run pytest --cov=$${{ inputs.codeCoverageProjectName }} --cov-fail-under=${{ inputs.codeCoverageFailUnder }}                                              ─╯poetry run pytest --cov=agent_management_system --cov-report term-missing -vv
```

Comment:
what is this `─╯` char??? lol

Reply:
ooh oops copy 🍝

Reply:
addressed: [c7f2ac1](https://github.com/daiseeai/workflows/pull/11/commits/c7f2ac1ecc4962f6a023c4e21f38f9ba2287bb9f)